<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaRecorder</class>
 <widget class="QWidget" name="MediaRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>7</number>
     </property>
     <widget class="QWidget" name="verticalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="QScrollArea_LeftContainer">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-135</y>
            <width>748</width>
            <height>851</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QGroupBox" name="QGroupBox_GeneralSettings">
             <property name="title">
              <string>General Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Select what to record</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_RecordingType">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>None</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Audio Only</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Video Only</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Both</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="QCheckBox_Minimize">
                <property name="text">
                 <string>Minimize application when recording starts</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="QGroupBox_AudioSettings">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Audio Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Select audio input device</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_InputDeviceAudio">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>No devices detected</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Select audio recording codec</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_FormatAudio">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Select audio file format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_AudioFormat">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>.acc (ACC)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>.wav (Wave)</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Output audio file name (Not used if audio is discarded)</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLineEdit" name="QLineEdit_AudioFileName">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>Enter file name...</string>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="QToolButton_AudioFileName">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Apply</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Output folder of final audio file (Not used if audio is discarded)</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0">
                <item>
                 <widget class="QLabel" name="QLabel_AudioOutputFilePath">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>No path</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="QPushButton_AudioOutputFilePath">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Change</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="QGroupBox_VideoSettings">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Video Settings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Select screen to record</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_InputDeviceVideo">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>No devices detected</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Video recording codec</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_FormatVideo">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Video recording format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_VideoFormat">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>.mkv (Matroska)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>.mp4 (MPEG4)</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Frame rate</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_FrameRate">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>30</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>60</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Recording quality</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="QComboBox_VideoQuality">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <item>
                   <property name="text">
                    <string>High</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Low</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Normal</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Very High</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Very Low</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Output video file name (Not used if video is discarded)</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLineEdit" name="QLineEdit_VideoFileName">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="placeholderText">
                   <string>Enter file name...</string>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="QToolButton_VideoFileName">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Apply</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Output folder of final video file (Not used if video is discarded)</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,0">
                <item>
                 <widget class="QLabel" name="QLabel_VideoOutputFilePath">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="text">
                   <string>No path</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="QPushButton_VideoOutputFilePath">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Change</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="QPushButton_Start">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Start Recording</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="QPushButton_Stop">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Stop Recording</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QVideoWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
